src/atsam/sam3_usb.c

```diff
@@ -216,8 +220,12 @@ usbserial_init(void)
         ;
 
     // Enable USB
+    #if CONFIG_MACH_SAM3
     UOTGHS->UOTGHS_CTRL = (UOTGHS_CTRL_UIMOD | UOTGHS_CTRL_OTGPADE
                            | UOTGHS_CTRL_USBE);
+    #elif CONFIG_MACH_SAME70Q
+    UOTGHS->UOTGHS_CTRL = (UOTGHS_CTRL_UIMOD | UOTGHS_CTRL_USBE);
+    #endif
```

Comment:
Can create a new `#define CFG_UOTGHS_CTRL` in existing `#if CONFIG_MACH_SAME70` block.